src/data/queries/audienceTopics.ts

```diff
@@ -3,14 +3,21 @@ import { ParagraphAudienceTopics } from '@/types/drupal/paragraph'
 import { QueryFormatter } from 'next-drupal-query'
 import { AudienceTopic } from '@/types/formatted/audienceTopics'
 
-const getTagsList = (fieldTags) => {
-  if (!fieldTags) return null
+interface Tag {
```

Comment:
is this needed?

if the formatter is returning `AudienceTopic[]` but it just calls getTagsList which returns `Tag[]`, can we re-use one or the other?

Reply:
Missed that, thanks